### DISCUSSION TITLE
See Response on S3-objects

### DISCUSSION THREAD

**Discussion by erwinmoretz** [thunderclient/thunder-client-support#1304] [Q&A]
Hi rangav, 

i have a problem to get a response, when i send a GET Request to a S3-Bucket.

![image](https://github.com/rangav/thunder-client-support/assets/112586589/7f7fafa2-55eb-425a-9fd0-50813d4d691e)

When I send the same request with the same URL in Postman, it is possible to get a valid response. 

Can you tell me how to access the response with TC? 

Thanks

**Comment by rangav** [comment]
Hi @erwinmoretz 

I haven't tested S3 URLs before. do you have a sample/test URL so I can test here?

**Reply by erwinmoretz** [reply]
it's not possible for me to share a link from our S3-Bucket, because of compliance. 

our request url looks like: http://[myhost]-local.s3.localhost:[port]/[object]
-> Response looks like my screenshot

we have also tried a url like this: http://[myhost]-local.s3.localhost.com
-> No Result like the above (couldnt resolve hostname) 
->Response Screenshot: 
![image](https://github.com/rangav/thunder-client-support/assets/112586589/7556ecb2-d2a9-4486-8e91-b7f88f93d200)

We suppose that Problem is on parsing, because of missing domain-name for example "**.com**"

Alternative: 
I think you have to mock the S3 Service locally. Maybe this App will help you https://s3ninja.net/

**Comment by erwinmoretz** [comment]
Hi rangav,

should i create an issue for this Problem?

**Reply by rangav** [reply]
Can you try the solution mentioned in this comment - https://github.com/rangav/thunder-client-support/issues/1086#issuecomment-1664276140

**Reply by erwinmoretz** [reply]
yes, i tried all three solutions [127.0.0.1 | ::1 | 0.0.0.0]. They dont work for me :/ 

i used them instead of my localhost. 

_http://[myhost]-local.s3.localhost:[port]/[object]_

**Reply by rangav** [reply]
Thanks for the confirmation. Please create a new issue.

Also please share any links on how to setup S3 locally.
